### PR TITLE
feat: profile activities page

### PR DESCRIPTION
**Pull request by MoMentalochka**
## Standards checklist:

- [ ] The PR title is descriptive.
- [ ] The PR doesn't replicate another PR which is already open.
- [ ] The PR has self-explained commits history.

- [ ] The code is mine or it's from somewhere with an Apache-2.0 compatible license.
- [ ] The code is efficient, to the best of my ability, and does not waste computer resources.
- [ ] The code is stable and I have tested it myself, to the best of my abilities.
- [ ] If the code introduces new classes, methods, I provide a valid use case for all of them.

## Changes:

- [...]

### CODE REVIEW

**Comment by shpingalet007** [comment]
Until node v21 is ready, we can't merge this PR

**Review by maxgithubprofile — APPROVED** [review]

**Review by shpingalet007 — APPROVED** [review]
Quality checked. There are some lintings, but it's fine...

Bastyon node v21 is released, can be merged now.

**Pull request merged**
